docs/01-app/03-api-reference/04-functions/setTagStale.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: setTagStale
+description: API Reference for the setTagStale function.
+---
+
+`setTagStale` allows you to mark [cached data](/docs/app/guides/caching) as stale for specific cache tags without immediately invalidating it.
+
+## Usage
+
+`setTagStale` can be called in Server Functions and Route Handlers.
+
+`setTagStale` cannot be called in Client Components or Middleware, as it only works in server environments.
+
+> **Good to know**: Unlike [`revalidateTag`](/docs/app/api-reference/functions/revalidateTag), `setTagStale` marks data as stale but preserves the existing timestamp under an `expire` field in the tag manifest. This allows for more granular cache control where data can be marked stale without immediate invalidation.
+
+## Parameters
+
+```tsx
+setTagStale(tags: string | string[]): void;
+```
+
+- `tags`: A string or array of strings representing the cache tags associated with the data you want to mark as stale. Each tag must not exceed 256 characters and values are case-sensitive.
+
+You can add tags to `fetch` as follows:
+
+```tsx
+fetch(url, { next: { tags: [...] } });
+```
+
+## Returns
+
+`setTagStale` does not return a value.
+
+## Relationship with `revalidateTag`
+
+`setTagStale` complements [`revalidateTag`](/docs/app/api-reference/functions/revalidateTag) by providing more granular cache control:
+
+- `setTagStale` marks data as stale while preserving the original timestamp under an `expire` field
```

Comment:
```suggestion
- `setTagStale` marks data as stale while preserving the original cache entry until the new data is updated, rather than purging the underlying cache entry.
```